lib/litle.rb

```diff
@@ -27,6 +27,14 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class LitleGateway < Gateway
+      def initialize(options={})
+        if options.has_key?(:secure_page_url)
+          self.test_url = options[:secure_page_url]
```

Comment:
`test_url` is different than the `secure_page_url`. The former is the API endpoint, not necessarily the same as the PayPage host.